setup.cfg

```diff
@@ -1,14 +1,72 @@
+[metadata]
+name = DecayLanguage
+description = A language to describe particle decays, and tools to work with them.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/scikit-hep/decaylanguage
+author = Henry Fredrick Schreiner III, Eduardo Rodrigues
+author_email = henry.schreiner@cern.ch, eduardo.rodrigues@cern.ch
+maintainer = The Scikit-HEP admins
+maintainer_email = scikit-hep-admins@googlegroups.com
+license = BSD-3-Clause
+license_file = LICENSE
+platforms = Any
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+keywords =
+    HEP
+    particle
+    decay
+    chain
+    representation
+
+[options]
+packages = find:
+install_requires =
+    attrs>=19.2
+    graphviz
+    lark-parser>=0.8.0,<0.8.6
+    numpy>=1.12
+    pandas>=0.22
+    particle==0.14.*
+    plumbum>=1.6.9
+    six>=1.11
+    cachetools;python_version<"3.3"
+    enum34>=1.1;python_version<"3.4"
+    importlib_resources>=1.0;python_version<"3.7"
+    pathlib2>=2.3;python_version<"3.5"
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+include_package_data = True
+
+[options.packages.find]
+exclude =
+    tests
+
+[options.extras_require]
+test =
+    pytest>=4.6
+
 [bdist_wheel]
 universal = 1
 
-[aliases]
-test=pytest
-
```

Comment:
Deprecated, users should not be calling `python setup.py <anything>`

Reply:
Fair.